docs/daily-notes.md

```diff
@@ -1,36 +1,49 @@
 # Daily notes
 
-The idea is to make it easier for people to be able to create Daily notes.
-
-This feature is open for discussion at [foambubble/foam#54](https://github.com/foambubble/foam/issues/54).
-
-## Must have
-
-- Should exist as part of [foam-vscode](https://github.com/foambubble/foam/tree/master/packages/foam-vscode) extension 
-- A new command "Foam: Open Daily Note", which creates new file and focuses into it, or opens the existing file for today if it exists already.
-  - File should have a Level 1 `# Heading`
-- Extension should define a hotkey for it (TBD? What do other apps do?)
-
-## Should have
-
-- Settings to customise:
-  - Format of file name, default to `yyyy-mm-dd`
-  - Format of title
-    - Default to same as file name
-    - Allow override e.g. `MMMMM D, YYYY`, to July 8. 2020 (locale specific)
-  - Extension of file, default to `.md`
-  - Directory into which file should be created, default to `./` (workspace root)
-    - Should create directory if needed
-
-## Could have
-- Automatically update a "daily notes" index file, newest first. Format TBD
-  - A custom note template .md file that could be stored in `.vscode/` directory (would supercede having to format the title)
-    - This could be useful for people who e.g. want there to be a format for every day's notes with fixed questions and stuff.
-  - Maybe we could do it with just back links or tags, by putting e.g. `[[daily]]` into the note template
-  - If files were named in alphabetic sortable order, and back links would display in reverse order, newest would always come on top
-
-[//begin]: # "Autogenerated link references for markdown compatibility"
-[todo]: todo "Todo"
-[roadmap]: roadmap "Roadmap"
-[//end]: # "Autogenerated link references"
+Automatically create a Daily Note by executing the "Foam: Open Daily Note" command . If a daily note for today's date already exists, the command opens the existing note.
+
+![Daily note feature in action](assets/images/daily-note.gif)
+
+## Keyboard shortcut
+
+The default keyboard shortcut for "Open Daily Note" is `alt`+`d`. This can be overridden using [VS Code Keybindings editor](https://code.visualstudio.com/docs/getstarted/keybindings).
```

Comment:
```suggestion
The default keyboard shortcut for "Open Daily Note" is `alt`+`d`. This can be overridden using the [VS Code Keybindings editor](https://code.visualstudio.com/docs/getstarted/keybindings).
```